conda_smithy/templates/azure-pipelines-osx.yml.tmpl

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: osx
   pool:
-    vmImage: macOS-10.13
+    vmImage: macOS-10.14
```

Comment:
Can this be `10.15`?

Reply:
If we are brave enough, then yes. It was made available by MS on Feb. 20, so it may be too new. I can update it if you wish.

Reply:
Maybe we should stay with the image that has been around longer?

Reply:
Either way is fine with me as a contributor. Up to you. I can revert my last commit 7c19364.

Reply:
> Maybe we should stay with the image that has been around longer?

At conda-forge, we don't really depend on anything from a macOS version. Just tools like bash, curl, etc.

Reply:
It's not so much that as this is a brand new image. I'm not sure how much testing it has received. Are we sure we are ready to use it across thousands of feedstocks?

Reply:
It works fine as @mrakitin said and with our compilers, we should be isolated from libraries outside.

Reply:
ping @jakirkham

Reply:
Reverted for now, since we are in need of this change ASAP.